_integrator-guide/jsonata-mapper-dates-times.md

```diff
@@ -0,0 +1,219 @@
+`$now()` functions as well. Therefore, this article will cover only the
+`$moment()` extension:
+
+*   [Basic usage](#basic-usage)
+*   [Formatting the outcome](#formatting-the-outcome)
+*   [Date and time operations](#date-and-time-operations)
+*   [Warning: Parsing Inconsistency pitfall](#warning-parsing-inconsistency-pitfall)
+
+## Basic usage
+
+Use `$moment()` to return the current timestamp (equivalent to `$now()`).
+For example, if the local time is 2017-09-01T12:00:00 (Germany Time, Central
+Europen Summer Time) which is 1-hour ahead of UTC + DST (Daylight Saving Time).
+
+```js
+$moment()
+```
+
+The output is going to be in ISO 8601 standard format:
+
+```
+2017-09-01T10:00:00.000Z
+```
+
+To convert the date explicitly to UTC use:
+
+```js
+$moment().utc()
+```
+
+This implementation will set the used dates to UTC. However, sometimes this might
+not be the most desirable solution especially when the timezone information is
+crucial for the operations.
+
+When a date string is passed:
+
+```js
+$moment('06.12.2017')
+```
+
+Since the passed date is ambiguous, the outcome can be unexpected as well:
+
+*   6st of December 2017 (DD.MM.YYYY) = `"2017-12-05T23:00:00.000Z"`. Converted to UTC and no DST due to the time of the year.
+*   12th of June 2017 (MM.DD.YYYY) = `"2017-06-11T22:00:00.000Z"`. Converted to UTC and + DST.
+
+In above case, it was assumed the German Time (Central Europen Time zone). It
+will get complicated if it is not, therefore, one should prevent this inconsistency
+in advance by using known [ISO 8601 standard](https://en.wikipedia.org/wiki/ISO_8601)
+or String + Format method:
+
+```js
+// String + format
+$moment('06.12.2017','DD.MM.YYYY')
+// or ISO 8601
+$moment('2017-12-06')
+```
+
+Both implementations will return the same value:
+
+```
+"2017-12-05T23:00:00.000Z"
+```
+
+## Formatting the outcome
+
+`$moment()` returns the date and time in a standard ISO 8601 format but it can
+be modified by using .format() parameter. When used in the initial empty state:
+
+```js
+$moment('2017-12-06').format() & "\n" &
+$moment('2017-06-12').format()
+```
+
+Would return:
+
+```
+2017-12-06T00:00:00+01:00
+2017-06-12T00:00:00+02:00
+```
+
+These are the ISO 8601 formatted date and the time values offset from UTC at
+each particular moment (German Time). It is also possible to output using many
+other formats. For example:
+
+```js
+$moment('2012-12-01').format('MM/DD/YYYY')
+```
+
+This expression returns the input ISO 8601 value in a formatted US date format.
+
+```
+12/01/2012
+```
+
+Let us take another example:
+
+```js
+$moment('2012-12-01T22:32:16').format("dddd, MMMM Do YYYY, h:mm:ss a")
+```
+
+This outcome of this expression would be:
+
+```
+Saturday, December 1st 2012, 10:32:16 pm
+```
+
+More details about possible formatting is available from the
+[Moment.js documentation pages on formatting](https://momentjs.com/docs/#/displaying/format/).
+
```

Comment:
Strange, are you sure you went through the document? There is a section, explicitly talking about **Formatting the outcome** where `.format( )` is talked about in some detail.

Reply:
Are you refering to the section that ends on this line?  Yes, I did see this and yes I saw the examples here.  I just believe that the two examples above are relevant enough to merit a mention in these examples since ISO 8601 to unix timestamps is a very common pattern.